test/system/static.test.ts

```diff
@@ -359,55 +357,6 @@ test("static analysis for distroless base-debian10", async (t) => {
   );
 });
 
-test("experimental static analysis for debian images", async (t) => {
```

Comment:
I'm wondering why we don't need this test anymore. 🤔

Reply:
Good spot! For two reasons:
- we don't have "experimental" static analysis flow, we have only 1 flow and that is "static analysis"
- we already have tests that check the default static analysis flow for both image identifier and image SHA (and archives)